src/Algorithms/cellpose_inference/config.yaml

```diff
@@ -0,0 +1,393 @@
+    default: 1.0
+    cli_tag: "--anisotropy"
+    optional: True
+    section_id: "algorithm-args"
+    mode: "advanced"
+  - name: exclude_on_edges
+    type: Checkbox
+    label: "Exclude On Edges"
+    description: "exclude cells on edges"
+    value: False
+    cli_tag: "--exclude_on_edges"
+    optional: True
+    section_id: "algorithm-args"
+    mode: "advanced"
+
+# Output Arguments
+  - name: save_png
+    type: Checkbox
+    label: "Save PNG"
+    description: "save segmentation as PNG"
+    value: False
+    cli_tag: "--save_png"
+    optional: True
+    section_id: "output-args"
+    mode: "beginner"
+  - name: save_tif
+    type: Checkbox
+    label: "Save TIF"
+    description: "save segmentation as TIF"
+    value: False
+    cli_tag: "--save_tif"
+    optional: True
+    section_id: "output-args"
+    mode: "beginner"
+  - name: no_npy
+    type: Checkbox
+    label: "No NPY"
+    description: "do not save npy"
+    value: False
+    cli_tag: "--no_npy"
+    optional: True
+    section_id: "output-args"
+    mode: "advanced"
+  - name: dir_above
+    type: Checkbox
+    label: "Dir Above"
+    description: "save output in directory above"
+    value: False
+    cli_tag: "--dir_above"
+    optional: True
+    section_id: "output-args" 
+    mode: "advanced"
+  - name: in_folders
+    type: Checkbox
+    label: "In Folders"
+    description: "save output in folders"
+    value: False
+    cli_tag: "--in_folders"
+    optional: True
+    section_id: "output-args"
+    mode: "advanced"
+  - name: save_flows  
+    type: Checkbox
+    label: "Save Flows"
+    description: "save flows"
+    value: False
+    cli_tag: "--save_flows"
+    optional: True
+    section_id: "output-args"
+    mode: "advanced"
+  - name: save_outlines
+    type: Checkbox
+    label: "Save Outlines"
+    description: "save outlines"
+    value: False
+    cli_tag: "--save_outlines"
+    optional: True
+    section_id: "output-args"
+    mode: "advanced"
+  - name: save_ncolor
+    type: Checkbox
+    label: "Save NColor"
+    description: "save ncolor"
+    value: False
+    cli_tag: "--save_ncolor"
+    optional: True
+    section_id: "output-args"
+    mode: "advanced"
+  - name: save_txt
+    type: Checkbox
+    label: "Save TXT"
+    description: "save txt"
+    value: False
+    cli_tag: "--save_txt"
+    optional: True
+    section_id: "output-args"
+    mode: "advanced"
+
+outputs:
+  - name: output_dir
+    type: Files
+    label: "Download Outputs"
+    description: "Directory containing the output files"
+    mode: "beginner"
+
+exec_function: 
+  name: "generate_cli_command"
+  script: "cellpose_inference"
+  module: "Algorithms.cellpose_inference"
+  cli_command: "python -m cellpose --verbose"
+  hidden_args: 
+    # dummy example
+    # - cli_tag: "--save_png"
+    #   value: True
+
+docker-image:
+  org: cellprofiler
+  name: runcellpose_no_pretrained
+  tag: 0.1
+
+folder_name: "cellpose_inference"
+
+citations:
```

Comment:
Individual algorithms should only have their own citations - interface templates should have THEIR citations, and either pull from a common bilayers citation file somewhere else OR we just repeat the bilayers citation across all interfaces (but that seems more annoying possibly)

Reply:
Noted